Gems/GradientSignal/Code/Source/Components/ImageGradientComponent.cpp

```diff
@@ -117,6 +234,11 @@ namespace GradientSignal
                 ->Field("TilingX", &ImageGradientConfig::m_tilingX)
                 ->Field("TilingY", &ImageGradientConfig::m_tilingY)
                 ->Field("StreamingImageAsset", &ImageGradientConfig::m_imageAsset)
+                ->Field("AdvancedMode", &ImageGradientConfig::m_advancedMode)
```

Comment:
I thought we only needed to bump the version if a field was deleted or we needed to handle some sort of conversion between merged fields? Also sadly the version is unused currently anyways :(

Reply:
I always thought we bumped it on _any_ change. Adding/deleting fields might be auto-handled correctly as long as you're good with default behavior, but it's still conceptually a change to the data we're storing, so ideally we would want to denote it that way. Another way to think about it is in the reverse - if you add a field, then load the data in an older version of the Editor, it would probably silently remove the field when it saves out the data, instead of warning you that you're trying to load a newer version of the data.

Reply:
That said, if the version number isn't used any more because of prefabs, well, I guess then it doesn't matter... :(

Reply:
Good point, yes it's not used anymore due to prefabs, but I will bump the version for posterity, because I think at some point there is a plan for versioning.